### ISSUE TITLE
False-positive InvalidConsoleOptionValue with named arguments

### ISSUE THREAD

**Issue by Etherlord** [psalm/psalm-plugin-symfony#179]
When passing `default` through named arguments, I get the following error

`ERROR: InvalidConsoleOptionValue - src/Console/DebugCommand.php:27:78 - Use Symfony\Component\Console\Input\InputOption constants
        $this->addOption('test', mode: InputOption::VALUE_REQUIRED, default: 'test');`

**Comment by seferov** [comment]
@Etherlord for addOption, mode is 3rd argument. Your code should be something like this:

```php
$this->addOption('test', null, InputOption::VALUE_REQUIRED, 'test');
```

**Comment by Etherlord** [comment]
@seferov I'm using php 8 named arguments. They allow you to skip optional parameters and pass values to a specific parameter by name. https://psalm.dev/r/cb0c002951 - example

**Comment by seferov** [comment]
@Etherlord I see now. Thanks for the report.
